db/migrate/20151101224210_generate_token_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokenForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
I created custom rake task as mentioned above. I'm not sure how to handle the migration now, I generally don't edit a migration once it's been run. In this case I could probably get away with it, but to follow best practices - how could I write a new migration to undo this? Would using the revert method be an option?